File: multimodule/module-api/src/main/java/com/gosuljo/api/hello/HelloController.java
package com.gosuljo.api.hello;

import com.gosuljo.common.entity.Member;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(value = "/hello")
public class HelloController {
    @GetMapping
    public String hello() {
        Member member = new Member();
        member.setId(1);
        member.setName("Hello~!");
        return member.toString();
    }
}

File: swagger/src/main/kotlin/com/gosuljo/swagger/hello/HelloController.kt
package com.gosuljo.swagger.hello

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class HelloController {
    @GetMapping("/hello")
    fun getHelloWorld(): ResponseEntity<String> {
        return ResponseEntity.ok("HelloWorld!")
    }
}